schema.sql

```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT GENERATED BY DEFAULT AS IDENTITY,
+    name varchar(100),
+    date_of_birth DATE,
+    escape_attempts INT,
+    neutered BOOLEAN,
+    weight_kg DECIMAL,
+    PRIMARY KEY(id)
 );
```

Comment:
- [ ] You should add a new column called `species` with a string type. For example:
`ALTER TABLE animals ADD species varchar(255);`

Reply:
Thank you 
![add species to animals table](https://github.com/Paulinelali/vet-clinic-project/assets/88007635/72cf1a87-bb60-4e0a-86f4-82509189a035)